### PR TITLE
Remove jquery

### PR DESCRIPTION
**Pull request by the-bionic**

### CODE REVIEW

File: addon/helpers/pikaday.js [hunk 1]
```diff
@@ -11,57 +10,61 @@ deprecate(
 );
 
 const openDatepicker = function(element) {
-  $(element).click();
+  document.querySelector(element).click();
```
Comment (efx) [inline addon/helpers/pikaday.js]: ` document.querySelector(element)` can return null. So we probably want to ensure this is a DOM element before trying to invoke `click()`.

File: addon/helpers/pikaday.js [hunk 2]
```diff
@@ -11,57 +10,61 @@ deprecate(
...
+    triggerNativeEvent(monthSelectorElements[0], 'change');
 
     triggerNativeEvent(
-      $(
-        'td[data-day="' +
-          day +
-          '"]:not(.is-outside-current-month) button:visible'
-      )[0],
+      document.querySelector(
+        `td[data-day="'${day}'"]:not(.is-outside-current-month) button:visible}`
```
Comment (efx) [inline addon/helpers/pikaday.js]: I think some of these are jQuery specific extensions we'll need to refactor / rethink.
I tried this quickly in Chrome and found the following:
Screen Shot 2020-02-06 at 15 30 35

File: addon/helpers/pikaday.js [hunk 3]
```diff
@@ -11,57 +10,61 @@ deprecate(
...
-    return $(this.selectorForYearSelect)
-      .children()
-      .last()
-      .val();
+    document.querySelector(YEAR_SELECTOR).lastChild.value;
   }
 };
 
+function updateElementValues(elements, value) {
+  Array.from(elements).forEach(function(e) {
```
Comment (efx) [inline addon/helpers/pikaday.js]: Great calling using `Array.from`! Not all browsers support this so we may need to document a breaking change and/or inform users somehow.

**Comment by efx** [comment]
Thank you for taking this on @the-bionic! Changes overall look good; we just need to figure out the migrations of some jQuery selectors. This PR has some examples, but a bit of trial and error testing might be quicker: https://github.com/adopted-ember-addons/ember-pikaday/pull/220/files

File: addon/helpers/pikaday.js (hunk 2 above)
Comment (the-bionic) [reply addon/helpers/pikaday.js]: This error is visible with jQuery too. 🤔

Comment (the-bionic) [reply addon/helpers/pikaday.js]: Something similar used here: https://github.com/adopted-ember-addons/ember-pikaday/blob/d0972f76a76dd1958cb625ad66f2109cc9dd2274/addon-test-support/interactor.js#L25

**Comment by the-bionic** [comment]
Ha! Thanks for the review and the pointers, I'll look to fix this soon. :)

File: addon/helpers/pikaday.js [hunk 4]
```diff
@@ -11,57 +11,64 @@ deprecate(
...
 const openDatepicker = function(element) {
-  $(element).click();
-
-  return PikadayInteractor;
+  const pickerElement = document.querySelector(element);
+  if (pickerElement) {
+    pickerElement.click();
+    return PikadayInteractor;
+  }
+  Ember.Logger.error(`${element} does not match any valid DOM element`);
```
Comment (efx) [inline addon/helpers/pikaday.js]: This is agreat idea to notify the user! I've read in an [RFC](https://emberjs.github.io/rfcs/0297-deprecate-ember-logger.html) that `Ember.Logger` will be deprecated at some point. Can we fail silently here? It begs the question of a more comprehensive error strategy, but I think that is outside the scope of this change.

**Comment by efx** [comment]
Looks great; thank you for doing this @the-bionic!
This rounds out #220.
